.github/workflows/test.yml

```diff
@@ -49,3 +49,17 @@ jobs:
       - name: Build
         run: cd Examples/topics && swift build
 
+  docs-examples:
+    runs-on: ubuntu-latest
+    container: swift:5.8
+    steps:
+    #   - uses: swift-actions/setup-swift@65540b95f51493d65f5e59e97dcef9629ddf11bf
+    #     with:
+    #         swift-version: ${{ matrix.swift }}
+      - name: Setup repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.github-token }}
+
+      - name: Test examples
```

Comment:
I know this file is already conflicting with the work you did in a different PR to consolidate the `examples` steps under a single job. You're going to want to do the same thing you did earlier here and just add another step to what is called `examples` in `main`, named something like`Build and run docs examples`. Now that I'm looking at the whole file from the `main` branch, I also think we can further consolidate and just move all of the `Build` steps from `examples` to the `test` job and remove the `examples ` job completely. My main motivation for wanting to keep these consolidated is avoiding the extra time needed to set up the container.